SpatialGDK/Source/SpatialGDK/Public/Utils/LatencyManager.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+
+#include <WorkerSDK/improbable/c_worker.h>
+
+#include "LatencyManager.generated.h"
+
+class USpatialNetConnection;
+class USpatialNetDriver;
+
+UCLASS()
```

Comment:
Why does this need to be a UClass? Isn't it only instantiated in one place and has it's lifecycle completely controlled?